stream_chat/tests/test_client.py

```diff
@@ -245,3 +240,17 @@ def test_update_blocklist(self, client):
 
     def test_delete_blocklist(self, client):
         client.delete_blocklist("Foo")
+
+    def test_normalize_sort(self, client):
+        expected = [
+            {"field": "field1", "direction": 1},
+            {"field": "field2", "direction": -1},
+        ]
+        actual = client.normalize_sort([{"field1": 1}, {"field2": -1}])
+        assert actual == expected
+        actual = client.normalize_sort(
+            [{"field": "field1", "direction": 1}, {"field": "field2", "direction": -1}]
+        )
+        assert actual == expected
+        actual = client.normalize_sort({"field1": 1})
```

Comment:
might be a good idea to provide multiple fields in one dict

Reply:
I added that. Let's see if tests fail